<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DhtWindow</class>
 <widget class="QMainWindow" name="DhtWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PeerNetQt: Dht Summary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <widget class="QLabel" name="dhtLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>DHT Details:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="queryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Query Details</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Dht Queries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="queryTreeWidget">
          <column>
           <property name="text">
            <string>PeerId</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Flags</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Results</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dht Bucket Entries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="dhtTreeWidget">
          <column>
           <property name="text">
            <string>Bucket</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ip:Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Key</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>StatusFlags</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Found</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Send</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Recv</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
